bucket/simple-dnscrypt.json

```
@@ -0,0 +1,63 @@
+{
+    "version": "0.7.1",
+    "description": "An easy to use GUI for dnscrypt-proxy",
+    "homepage": "https://simplednscrypt.org/",
+    "license": "MIT",
+    "suggest": {
+        "vcredist2015-2022": "extras/vcredist2022"
+    },
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/bitbeans/SimpleDnsCrypt/releases/download/0.7.1/SimpleDNSCrypt64.msi",
+            "hash": "5BA0C94774ACD7ADE18446B4351E7D28A948C490AEFC50DC6C08C13551F19F7F"
+        },
+        "32bit": {
+            "url": "https://github.com/bitbeans/SimpleDnsCrypt/releases/download/0.7.1/SimpleDNSCrypt.msi",
+            "hash": "CAC6B8119AF3DA70A5977A2E73EE5214303990FEDD7ABBC5F247A1E2CF19A663"
+        }
+    },
+    "pre_install": [
+        "if (!(Test-Path \"$env:LOCALAPPDATA\\SimpleDnsCrypt\")) {New-Item -Path \"$env:LOCALAPPDATA\" -Name \"SimpleDnsCrypt\" -ItemType \"directory\" -Force | Out-Null}",
+        "New-Item \"$dir\\dnscrypt-proxy\\blacklist.txt\" -ItemType \"file\" | Out-Null",
+        "New-Item \"$dir\\dnscrypt-proxy\\blocked.log\" -ItemType \"file\" | Out-Null",
+        "New-Item \"$dir\\dnscrypt-proxy\\query.log\" -ItemType \"file\" | Out-Null",
+        "New-Item \"$dir\\dnscrypt-proxy\\domain-blacklist.txt\" -ItemType \"file\" | Out-Null",
+        "New-Item \"$dir\\dnscrypt-proxy\\domain-whitelist.txt\" -ItemType \"file\" | Out-Null"
```

```suggestion
        "'blacklist.txt', 'blocked.log', 'query.log', 'domain-blacklist.txt', 'domain-whitelist.txt' | ForEach-Object {",
        "    if (!(Test-Path \"$persist_dir\\$_\")) { New-Item \"$dir\\$_\" | Out-Null }",
        "}"
```
For `New-Item`, `ItemType` is `File` by default.

---

Looks good